src/gui/src/components/explorer-grid/selected-item/context.tsx

```diff
@@ -48,59 +54,103 @@ const getSocketInsights = (
     .filter(Boolean) as SocketSecurityDetails[]
 }
 
-export interface SelectedItemContextValue {
+type SelectedItemStoreState = {
```

Comment:
```suggestion
type SelectedItemStoreState = DetailsInfo & {
```

see the comment below